framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+                timeStepList.append(line.split(',')[0])
+                depLabelList.append('dep'+'|'+materialList[1]+'|'+self.isotopeList[i])
+                depList.append(line.split(',')[i])
+    return depLabelList, depList, timeStepList, matList
+  
+  def getDepInfoMrtau(self,timeStepIndex,matchedTimeSteps):
+    """
+      Reads the MRTAU csv file to get the material density info relative to depletion.
+      @ In, timeStepIndex, integer, timestep number
+      @ In, matchedTimeSteps, list, list of time steps considered
+      @ Out, depLabelList, depList
+    """
+    depList = []
+    with open(os.path.join(self.workingDir,self.mrtauOutputFileMPI[0]), 'r') as outfile:
+      for line in outfile:
+        line = self.removeSpaces(line)
+        stringIsFloatNumber = self.isFloatNumber(line.split(','))
+        if stringIsFloatNumber is True:
+          if (float(line.split(',')[0]) == float(matchedTimeSteps[timeStepIndex])):
+            line = re.sub(r'\n',r'',line)
+            for i in xrange (0,len(self.isotopeListMrtau)):
+              depList.append(line.split(',')[i+1])
+    return depList
+    
+  def findDecayHeat(self,line):
+    """
+      Determines whether the decay heat is printed.
+      @ In, line, string
+      @ Out, findDecayHeat, boolean, True if decay heat is printed, false otherwise
+      @ Out, DecayHeatUnits, string
+    """
+    DecayHeatUnits = None
+    isDecayHeatPrinted = line.split(' ')[-2]
+    if isDecayHeatPrinted == 'YES':
+      DecayHeatUnits =  line.split(' ')[-1]
+      return True, DecayHeatUnits
+    else:
+      return False, DecayHeatUnits
+  
+  def numberOfMediaUsed(self,mpi,numberOfMPI):
+    """
+      Finds the number of media used in a given INSTANT output.
+      @ In, mpi, integer, MPI number considered
+      @ In, numberOfMPI, integer, total number of MPI user-selected
+      @ Out, mediaUsed, integer, number of media treated in one INSTANT MPI output
+    """
+    count = 0
+    if self.phisicsRelap is True and mpi >= numberOfMPI - 1:
+      pass
+    else:
+      with open(os.path.join(self.workingDir, self.instantOutputFileMPI[mpi])) as outfile:
+        for line in outfile:
+          if re.search(r'Medium\s+\d+\s+used',line):
+            count = count + 1
+    return count
+
+  def getCPUtime(self,numberOfMPI):
+    """
+      Gets the PHISICS CPU time.
+      @ In, numberOfMPI, interger, number of MPI user-selected
+      @ Out, cpuTime, list, cpu time (string) in a list
+    """
+    cpuTimes = []
+    for mpi in xrange (0,numberOfMPI):
+      with open(os.path.join(self.workingDir,self.instantOutputFileMPI[mpi])) as outfile:
+        for line in outfile:
+          if re.search(r'CPU\s+time\s+\(min\)',line) and self.phisicsRelap is False:
+            cpuTimes.append(line.strip().split(' ')[-1])
+          if self.phisicsRelap is True:
+            return 'na'
+    cpuTime = self.summedListValues(cpuTimes)
+    return [cpuTime]
```

Comment:
?

Reply:
Two things here: 
```
if self.phisicsRelap:
            return 'na'
```
return n/a if the coupled phisics/relap mode is on, because the cpu time is already calculated by RELAP. 
Also, I still need a cputime from phisics to be consistent with phisics standalone. So I just print a dummy. (I do not want to return 0. to avoid misleading results). 

As for `[cpuTime]`, the values printed in the csv file need to be in a list. So I put cpuTime right away in a list.